azure-kusto-data/azure/kusto/data/kusto_trusted_endpoints.py

```diff
@@ -103,6 +101,7 @@ def set_override_policy(self, matcher):
         self._override_matcher = matcher
 
 
-_filename = os.path.join(os.path.split(os.path.abspath(__file__))[0], "wellKnownKustoEndpoints.json")
-_well_known_kusto_endpoints_data = json.load(open(_filename))
+_filename = Path(__file__).absolute().parent / "wellKnownKustoEndpoints.json"
```

Comment:
I'm unfamiliar with this syntax, but this seems off to me, and I'm also concerned about platform independence. Can you use a `join()` function instead?

Reply:
It's identical to join - https://docs.python.org/3.7/library/pathlib.html#operators

I can use the joinpath() method if you prefer though.

Reply:
If anyone else is like me, I do think `joinpath()` would be clearer.

Approving (with this suggestion)